src/data_model/eng2math.jl

```diff
@@ -126,12 +126,14 @@ See [`make_per_unit!`](@ref make_per_unit!) for further explanation.
 function transform_data_model(
     data::Dict{String,<:Any};
     kron_reduced::Bool=true,
+    phase_projected::Bool=true,
     multinetwork::Bool=false,
     global_keys::Set{String}=Set{String}(),
     eng2math_passthrough::Dict{String,<:Vector{<:String}}=Dict{String,Vector{String}}(),
     eng2math_extensions::Vector{<:Function}=Function[],
     make_pu::Bool=true,
     make_pu_extensions::Vector{<:Function}=Function[],
+    transformations::Vector{<:Function}=Function[],
```

Comment:
what is the use case for this that is not included in eng2math_extensions? If we can clearly define where this is different from the other, we should document it in the docstring

Reply:
I think `eng2math_extensions` might indeed cover this functionality, with some changes to the signature of add_vrvi! however. I will remove it for now, and come back later to it if it does not cover the use case completely.